### PR TITLE
Picmi mcc

### PR DESCRIPTION
**Pull request by roelof-groenewald**
Functionality to run MCC from a python run script.

### CODE REVIEW

File: mewarpx/examples/capacitive_discharge.py [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+print('Setting up simulation with')
+print('  dt = %.3e s' % DT)
+print('  Total time = %.3e s (%i timesteps)' % (TOTAL_TIME, max_steps))
+print('  Diag time = %.3e s (%i timesteps)' % (DIAG_INTERVAL, diag_steps))
+
+##########################
+# physics components
+##########################
+
+v_rms_elec = np.sqrt(minutil.kb_J * T_ELEC / minutil.m_e)
```
Comment (peterscherpelz) [inline mewarpx/examples/capacitive_discharge.py]: @roelof-groenewald Please replace with WarpX / non-minutil constants and remove that import.

**Review by peterscherpelz — CHANGES_REQUESTED** [review]
1 small change. Thanks!

**Review by peterscherpelz — APPROVED** [review]

File: Python/pywarpx/picmi.py [hunk 2]
```diff
@@ -714,6 +714,39 @@ def initialize_inputs(self):
...
+
+        collision.scattering_processes = self.scattering_processes.keys()
+        for process, kw in self.scattering_processes.items():
+            for key, val in kw.items():
+                if key == 'species':
+                    val = val.name
+                collision.add_new_attr(process+'_'+key, val)
+
+        pywarpx.Collisions.collisions_list.append(collision)
+
```
Comment (nruof) [inline Python/pywarpx/picmi.py]: Why does the newcollision function append to "collisions_list" when the "collision" also gets appended in the last line of 
"initialize_inputs" in "MCCCollisions"?

**Review by EtherealMC-Bit — COMMENTED** [review]
I don't see any problems other than what Nich pointed out

**Comment by roelof-groenewald** [comment]
Thanks @nruof, nice catch!

**Comment by peterscherpelz** [comment]
@nruof @MKieburtz Can one of you leave an approving review then? You can then merge it.

**Review by EtherealMC-Bit — APPROVED** [review]
Looks good

**Pull request merged**